raiden_installer/constants.py

```diff
@@ -0,0 +1,31 @@
+import sys
+from pathlib import Path
+from xdg import XDG_DATA_HOME, XDG_CONFIG_HOME
+
+
+'''
```

Comment:
This would fit better as a module docstring, which should be at the top of the file.